CONTRIBUTING.md

```diff
@@ -79,3 +79,44 @@ To unify the appearance of all commit messages we only accept commit messages us
 -   Name the PR like its branch name (e.x. `Tech/123/my branch name`). The name is usually created by GitHub automatically
 -   Add the correct labels
 -   The PR Assignee is only used by the reviewer to see who is reviewing it
+
+## Changelog Guidelines
+
+###Why should you we write a changelog?
+
+-   A changelog is vital for the developers to keep track of their work. But most importantly now that it appears on every version update to every user, we should make sure that it is also user-friendly. And for that, we need guidelines to help us.
+
+###When should you add to the changelog file?
+
+-   You should only add a changelog entry after the merge (or approval) of your feature/fix not before.
```

Comment:
Normally, we develop new features and/or bug fixes in separate feature branches. So we can add the changelog entry directly in that branch. Only after the merge of the feature, the changelog entry will appear in the main branch.

Reply:
I realized that after writing it. I fixed it.